.gitignore

```diff
@@ -2,6 +2,7 @@
 logs
 project/project
 project/target
+app/assets
```

Comment:
did you mean to add this?

Reply:
Yeah, it was upsetting me that I couldn't do `git add .` to add all my changed files to a commit because it would track all of the assets

Reply:
fair enough